### PR TITLE
small style tweaks

### PR DESCRIPTION
**Pull request by ccsouthard**

### CODE REVIEW

File: assets/css/index.styl [hunk 1]
```diff
@@ -393,6 +405,11 @@ footer
 #vendor-list
   padding: 0
   list-style-type: none
+  line-height: 1.8;
+  text-decoration: none;
+  color: #707070;
+  font-weight: normal;
+}
```
Comment (notslang) [inline assets/css/index.styl:412]: that won't compile

File: assets/css/index.styl [hunk 2]
```diff
@@ -35,6 +35,13 @@ h1
 h2
   font-size: 1.5em
   margin: 0.83em 0
+  font-size: 22px;
+  line-height: 22px;
+  letter-spacing: 4px;
+  margin-bottom: 10px;
+  font-weight: 400;
+  text-transform: uppercase;
+  text-align: center;
```
Comment (notslang) [inline assets/css/index.styl:44]: Are you sure you want all the h2's centered? That doesn't match the concept you sent me from https://obeygiant.com/fine-art/

Comment (ccsouthard) [reply assets/css/index.styl:44]: Good eye. I should have specified the H2's.  it doesn't work on this page. Do you think that the larger font competes with the logo or that it works.

Comment (notslang) [reply assets/css/index.styl:44]: It doesn't really compete with the logo, but the letter spacing is weird. The spaces are so big it's almost the size of spaces between the words.

File: assets/css/index.styl (hunk 1 above)
Comment (ccsouthard) [reply assets/css/index.styl:412]: why not?

**Pull request closed without merging**